content/600-about/200-prisma-docs/20-style-guide/03-spelling-punctuation-formatting.mdx

```diff
@@ -184,16 +188,17 @@ Use the following as reference when creating and editing docs: [formatting inlin
 
 Placeholders are a tricky topic in technical documentation and are one of the most common sources of confusion, especially for beginners. To strive for consistency, placeholders in the Prisma docs:
 
-- are spelled in all-uppercase letters
-- are prefixed and suffixed with two underscores
+- are spelled in all-lowercase letters
+- are prefixed and suffixed with an angle bracket
+- use hyphens to join multipe words
 - use descriptive terms
 
-As an example, consider the following code block where `__DATABASE_CONNECTION_URL__` is a placeholder for the PostgreSQL connection URL:
+As an example, consider the following code block where `<database-connection-url>` is a placeholder for the PostgreSQL connection URL:
 
 ```prisma
 datasource db {
   provider = "postgresql"
-  url      = "__DATABASE_CONNECTION_STRING__"
+  url      = "<database-connection-url>"
```

Comment:
This is quite a change from the current pattern. Is there a reason behind this?
Using `<...>` is often discouraged as it looks like a html tag - which the placeholder information almost never is.

Reply:
I was agnostic about this. @zachtil @keerlu  - would you like to comment?

Reply:
We didn't come to an agreement, so I've reverted this change

Reply:
We never resolved this, so I've reverted the change